src/main/java/io/stargate/sgv2/jsonapi/api/model/command/clause/update/UpdateTarget.java

```diff
@@ -0,0 +1,52 @@
+package io.stargate.sgv2.jsonapi.api.model.command.clause.update;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.node.ArrayNode;
+import com.fasterxml.jackson.databind.node.ObjectNode;
+
+/**
+ * Definition of a target for an {@link UpdateOperation}; built from "dot path" and document to
+ * update, by {@link UpdateTargetLocator}.
+ */
+public record UpdateTarget(
+    String fullPath, JsonNode contextNode, JsonNode valueNode, String lastProperty, int lastIndex) {
```

Comment:
Can you add java doc not able to follow what is lastProperty and lastIndex for?

Reply:
Will do.